src/tests/evas/evas_test_image.c

```diff
@@ -904,7 +904,8 @@ EFL_START_TEST(evas_object_image_map_unmap)
 
    // save file, verify its pixels
    fd = eina_file_mkstemp("/tmp/evas-test.XXXXXX.png", &tmp);
-   close(fd);
+   fail_if(fd <= 0);
+   fail_if(close(fd));
```

Comment:
Don't we need `pipe_close` here?

Reply:
But...is this a pipe?

Reply:
welp, make sense haha. I thought we should use `eina_pipe` on every `write`, `read`, `close` for some reason

Reply:
Now that you mention...it is a `close` and not a `fclose`. Hm!

Reply:
Have you looked about this?

Reply:
Yes, it should be a `pipe_close`.

Reply:
oh, okay yays